### DISCUSSION TITLE
is normalization MUST for integration?

### DISCUSSION THREAD

**Discussion by complexgenome** [satijalab/seurat#4769] [Q&A] [answered]
Hi satija lab,

I have single cell data from different subjects. I performed preliminary quality control on them. I'd like to integrate them.
The final integrated seurat object needs to have subjects with different cell types and genes with _raw counts_.

I saw tutorials at: 
https://satijalab.org/seurat/articles/integration_introduction.html#introduction-to-scrna-seq-integration-1
https://satijalab.org/seurat/archive/v3.2/immune_alignment.html
https://satijalab.org/seurat/archive/v3.0/integration.html

All these tutorials include step to **normalize** data before performing the integration.

How can I proceed to achieve the desired integration with final output as raw count matrix object?

thank you.

**Comment by timoast** [comment, marked as answer]
Yes, you need to normalize the data before running integration. I'm not sure what your goal really is here...if you just need the raw count matrix for each object then there's no need to do integration. If you're looking for some version of a "corrected" count matrix, that's not something that we compute with the Seurat integration algorithm.

**Reply by complexgenome** [reply]
@timoast thanks for your reply.

I would like to perform deconvolution using single-cell data from multiple subjects.  I have single cell data for eight different subjects for a specific tissues region. Per subject one .mtx file 

Softwares (music, bisqueRNA, SCDC) for deconvolution want  raw counts from different subjects as an input to begin with. They want one file as an input. Whereas I have one object per subject, totaling to eight. 

Merging is little sketchy as the count of cells add up as the objects are put together. For instance subj1 has 20K genes and 19K cells, subj2 with 21K genes and 15K cells. 

I am unsure what to do and how to proceed at this point. 

Thank you again.

**Reply by timoast** [reply]
I think this is a question to ask the developers of those specific methods (music, etc), about how they handle data from multiple samples.
